packages/react/src/components/Tabs/Tabs.js

```diff
@@ -282,42 +364,66 @@ export default class Tabs extends React.Component {
       );
     });
 
+    const leftOverflowNavButtonHidden =
+      !this.state.horizontalOverflow || !this.state.tablistScrollLeft;
+    const rightOverflowNavButtonHidden =
+      !this.state.horizontalOverflow ||
+      this.state.tablistScrollLeft + this.state.tablistClientWidth ===
+        this.state.tablistScrollWidth;
```

Comment:
Does using scrollLeft and the calculations here work in all browsers when the UI is hosted in a RTL environment (dir="rtl")? I know from bitter experience that scrollLeft is reported differently depending on the browser being used (https://bugs.chromium.org/p/chromium/issues/detail?id=721759).

Reply:
yes RTL environments should be accounted for already according to the spec (also mentioned in the issue description you linked), so that seems to have been a Chromium specific bug

Reply:
Ok great, as long as you've checked. I just know that browsers don't report scrollLeft consistently when the dir attribute of a containing element further up in the DOM is set to "rtl", that's why this (https://www.npmjs.com/package/normalize-scroll-left) package exists. Your code above hides the leading nav button when there is no overflow or the scrollLeft is falsy (i.e. 0). The table in the package I linked to indicates that you won't get a falsy value for scrollLeft consistently across browsers.

Reply:
if you have a replicable test case please share it, but testing under the MDN [RTL guidelines](https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/RTL_Guidelines) didn't seem to run into the scrollLeft value issue

Reply:
Here is a repro case - https://codesandbox.io/s/blissful-golick-pvuo0?file=/index.js. Sorry it's taken me a long time to get round to doing this.